config/deployments/production.toml

```diff
@@ -70,6 +70,7 @@ noon.key_mode = "Live"
 nuvei.base_url = "https://ppp-test.nuvei.com/"
 opayo.base_url = "https://pi-live.sagepay.com/"
 opennode.base_url = "https://api.opennode.com"
+paybox.base_url = "https://preprod-ppps.paybox.com/PPPS.php"
```

Comment:
Can we please use the production url here?

Reply:
Yeah bro, merchant yet to provide production url

Reply:
If this is going to be merchant specific, shouldn't we be ideally using a templated URL like we've done for Adyen, for instance?

https://github.com/juspay/hyperswitch/blob/540ef071cb238a56d52d06687226aab7fd0dfe68/config/deployments/production.toml#L26